### ISSUE TITLE
av changes

### ISSUE THREAD

**Issue by radare** [radare/radare2#13229]
```
[0x1000011ec]> av?
Usage: av[?jr*]   C++ vtables and RTTI
| av                search for vtables in data sections and show results
| avj               like av, but as json
| av*               like av, but as r2 commands
| avr[j@addr]       try to parse RTTI at vtable addr (see anal.cpp.abi)
| avra[j]           search for vtables and try to parse RTTI at each of them
| avrr              recover class info from all findable RTTI (see aC)
| avrD [classname]  demangle a class name from RTTI
[0x1000011ec]> aC
Usage: aC[e] [addr-of-call] # analyze call args (aCe does esil emulation with abte)
[0x1000011ec]>
```

note the following:

* avra should be renamed to aavt (analyze all virtual tables)
* the aC command reference is wrong, it is ac
* avrr may be renamed to aavr maybe :?
* Is avra run from aaa? may we want to use the bin info for this?

**Comment by thestr4ng3r** [comment]
"aavt (analyze all virtual tables)" is wrong, the command is about rtti, not just vtables, also it doesn't really analyze, it just reads data and prints it. Does that fit under aa? avrr does in any case though.

**Comment by radare** [comment]
then i misunderstood. maybe is avrr the one that must be moved into aa? if those vtables thing are just printing commands why are they under analysis? PD: never used that

**Comment by thestr4ng3r** [comment]
Well where would it fit other than anal? It's partially heuristic search and partially parsing data structures defined by C++ ABI.

**Comment by radare** [comment]
Is av* executed from aaa? Which are the requirements for it to work? Can you fix the aC help message?

**Comment by thestr4ng3r** [comment]
Right now there is nothing in aaa, but avrr is something could certainly be added. The requirement for it are xrefs for the vtable search (aar has to be executed before) and rtti in the binary. Maybe there should also be a fallback when there is no rtti, so it just detects vtables and adds classes for them.

Yes, I'll fix the help

**Comment by radare** [comment]
Cc @maijin

Pls add this in aaa when it matters

**Comment by Maijin** [comment]
@thestr4ng3r Do you have an example of binary in r2r I can use to test the aaa integration?

**Comment by thestr4ng3r** [comment]
Not in r2r, but you can try with this: [cpp_sample.exe.gz](https://github.com/radare/radare2/files/2999628/cpp_sample.exe.gz)

**Comment by thestr4ng3r** [comment]
Here's also the pdb for it: 
[cpp_sample.pdb.gz](https://github.com/radare/radare2/files/2999629/cpp_sample.pdb.gz)

**Comment by Maijin** [comment]
@thestr4ng3r 

```
[0x14000100f]> avrr
No virtual tables found
```

Huh?

**Comment by thestr4ng3r** [comment]
Needs aar first

**Comment by Maijin** [comment]
ok great!

**Comment by radare** [comment]
@oddcoder ^ we need to know when an specific analysis has been run already because some depend on others like aao or avrr

**Comment by radare** [comment]
@thestr4ng3r any comments on the other points?

**Comment by thestr4ng3r** [comment]
The aC ref is fixed, for the rest my comments above are still valid

**Comment by Maijin** [comment]
I don't have any opinion on the renaming, won't have impact on the users I think because very few users know about it if not only core so feel free to go ahead :)

**Comment by radare** [comment]
removing the milesstone, and imho few users know it because most ppl expect things that analyze the whole thing to be under `aa`. we can also add some alias commands instead of renaming, so not breaking things, what do u think?

**Comment by thestr4ng3r** [comment]
I don't think we need an alias for this. A fitting name would be `aavr`, but that kind of clashes with `aav`, so not too sure about it.
